rotkehlchen/rotkehlchen.py

```diff
@@ -797,7 +797,8 @@ def query_balances(
                 force_token_detection=ignore_cache,
                 ignore_cache=ignore_cache,
             )
-            balances[str(Location.BLOCKCHAIN)] = blockchain_result.totals.assets
+            if len(blockchain_result.totals.assets):
+                balances[str(Location.BLOCKCHAIN)] = blockchain_result.totals.assets
```

Comment:
We need to ask @kelsos here. Kelso you don't read the all balance query, right? So in the case of no blockchain balances the blockchain key missing is fine in that endpoint, right?

Reply:
All balance is the one that creates the snapshot right? if yes, then that is correct the frontend ignores the result of this query.

Reply:
thank you kelsos